frontend/src/views/AccountView.vue

```diff
@@ -0,0 +1,43 @@
+<template>
+  <div class="flex-grow text-center">
+    <div class="text-green-500 italic" v-if="showerror">
+      Account nicht verfügbar
+    </div>
+  </div>
+  <div class="w-full mt-auto mb-auto ml-3 mr-3">
+    <AccountDetailsEditContainer
+      :showadminview="true"
+      :user="originalUser"
+      v-if="!showerror"
+    />
+  </div>
+</template>
+
+<script setup lang="ts">
+import {
+  LoernwerkError,
+  LoernwerkErrorCodes,
+} from '../../../backend/loernwerkError';
+import { IUser } from '../../../model/user/IUser';
+import AccountDetailsEditContainer from '../components/AccountDetailsEditContainer.vue';
+import { AccountRestInterface } from '../restInterfaces/AccountRestInterface';
+import { router } from '../router';
+import { ref } from 'vue';
+
+const showerror = ref(false);
+let originalUser: Partial<IUser>;
+try {
+  originalUser = await AccountRestInterface.getOwnAccount();
+  console.log(originalUser);
+} catch (e) {
+  if (e instanceof LoernwerkError) {
+    if (e.code === LoernwerkErrorCodes.UNAUTHORIZED) {
```

Comment:
We could implement a utiliy function for this sort of error handing, since some reactions might be used on all pages

Reply:
im not sure if we can get a advantage from this, because we may need to do diverse things on errors, the only common action (i currently could imagine) would be a redirect on a unauthorized error

Reply:
Status update: We did not talk about this, but i think i can come up with a shorter solution soon:tm:

Reply:
Made this a bit shorter. I would agree to do a utility function for the following part, since it would be occure in more classes: 
```ts
if (e instanceof LoernwerkError) {
 if (e.code === LoernwerkErrorCodes.UNAUTHORIZED) {
  router.push('LogIn');
 }
}
```
where should it be placed?

Reply:
Update: The NavigationBar will already catch not-logged-in users, since it does need to query their data to see if it should display the admin button and will redirect to login if 401 is encountered, so I'm not sure if we need to still detect it everywhere

Reply:
that's great i will remove it